Dockerfile

```diff
@@ -47,7 +47,14 @@ ARG RUNTIME_PACKAGES="\
   liblept5 \
   libpq5 \
   libxml2 \
+  liblcms2-2 \
```

Comment:
These might be bundled with the Pillow wheel, and not actually needed.  Something to test once I figure out how